.circleci/config.yml

```diff
@@ -38,20 +46,57 @@ defaults: &defaults
 
 version: 2
```

Comment:
This means we're using `circleci v2`?

Reply:
@kevinbarabash Yes, we're using CircleCI 2.0.